prototype_template/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Template project."""
+
+from .template_module import TemplateClass
```

Comment:
Was moving the code to `template_module` strictly necessary, or were you just looking for a way to demonstrate documentation at the module level?

Reply:
Just for demonstration, anyway everybody will spread their code into modules (I hope 🤞 ).